### ISSUE TITLE
No image on Gamecube System Menu

### ISSUE THREAD

**Issue by AlucardRetroChallenge** [ikorb/gcvideo#93]
Hello! I have a Brazilian DOL-002 gamecube (digital Port + SP2), modded with a Qoob Pro.
I use the Aliexpress version of GCPlug (the blue One), flashed with the ver. 3.1. (it came with 3.0d).
When I use interlaced resolution on Qoob Pro or Swiss via GC Plug, everything is fine, but when I access the original system menu, I don't get any image, just the menu sound.
And it happens with my capture Card (elgato hd60) and my 4k TV (Samsung KS8500 / KS7500).
The older 2011 model (32") doesn't get imagem at ALL, even forcing 480p via Qoob menu ir Swiss.
Thanks!

**Comment by ikorb** [comment]
Does this also happen with default settings (linedoubling enabled for all modes, enhanced DVI off)?

**Comment by AlucardRetroChallenge** [comment]
Yeah, It happens! To make it easier, I've recorded a direct feed footage. Qoob Menu, Swiss and game in 480i are ok, but nothing happens on original bios.
Please, take a look: https://1drv.ms/u/s!AsRv502u3B9niKl7qQqx2LITtLjBjg?e=aWfKUA
By the way, it also happend in ver. 3.0d (before the update).

**Comment by ikorb** [comment]
Does anything change if you disable "Fix Resolution" and "Fix Sync Timing" in the Advanced submenu?

**Comment by AlucardRetroChallenge** [comment]
![image](https://user-images.githubusercontent.com/107833265/176702323-3c8156ca-e964-4073-83a1-1b6a40d103a8.png)

Thanks for the answer! I have tried these settings, with every Digital Color Format, with linedoubling enabled for all modes, enhanced DVI off, and nothing happens. Have you ever seen this before?

**Comment by AlucardRetroChallenge** [comment]
Man... I've been thinking. Follow me to see if we can get through it.

My HDMI adapter is this. A supposely normal GCPlug. It came with the GCVideo 3.0d firmware. Please, take a look:

![image](https://user-images.githubusercontent.com/107833265/176793500-0e95cff9-8a8e-4e4a-a397-fc72ed087acb.png)

I've opened it, and saw this chip:

![image](https://user-images.githubusercontent.com/107833265/176793646-6285f139-5a93-4375-ac5b-25ce44262d28.png)

Ok. I've tried to update it with the "updater-3.1-gc.dol" file, with the "gcvideo-3.1-gcplug.zip" and it didn't work out, but, with the file "GCHD_MK2Rev1_3.1.zip", it was "good to go", as you could see in the video posted above.

I'm suspecting this GCHD update on this GCPlug of mine is not working properly. But then... I remember that the problem related to the topic was happening on the older firmware.

But (again, lol), I remember that once, a long time ago, on a old Samsung LED TV, with the old firmware (3.0d), it was possible to see the original bios / system menu with 480i resolution for a moment.

Said all of this, here comes my questions:

1) Is it possible that this GCPlug is a defective unit?

2) Assuming it is not, is it possible to have some kind of problem on my Gamecube motherboard? Is there any point on it to check - with a multimeter - if the 480i resolution is working properly through Digital Port? This resolution works fine on the composite cable.
I don't belive very much in this second alternative, because it is possible to output the 480i signal via HDMI on Qoob Menu and Swiss.

3) Last Question: based on the GCPlug chip shown on the picture, is there another update file compatible withi it other than the "GCHD_MK2Rev1_3.1.zip"? It quite obvious that my device is not that one, lol. This firmware on my GCPlug fells more like a "workaround".

Sorry for the long post and the poor english. I hope we can solve this mistery together.

Thanks! ^^

**Comment by Extrems** [comment]
Your unit isn't defective. This is an issue with PAL-M I've known of for years, but figured it couldn't be addressed.

**Comment by ikorb** [comment]
Oh, PAL-M. I only have access to Euro-PAL cubes (and one US-NTSC), so that case was never tested.

I wonder why Swiss calls it DOL-002 though, isn't that the code for the power supply?

**Comment by Extrems** [comment]
Good news, the NTSC video encoder supports PAL-M, and it's signaled the same way as PAL 50Hz is, which is the problem.

It is the part number for the power adapter, but for whatever reason, they labeled this system like so.

**Comment by Extrems** [comment]
You can use Game Boy Interface or Enhanced mGBA with `--format=pal-m` to test this, and also `--format=ntsc-50` for the reverse problem. It's also possible to use Not64 with `VideoFormat = 2`.

**Comment by AlucardRetroChallenge** [comment]
> Your unit isn't defective. This is an issue with PAL-M I've known of for years, but figured it couldn't be addressed.

Really? The swiss, on my Dol-002 Gamecube uses to work fine in every scenario that we've talked earlier.
When the console it's connected to my CRT via composite cable, swiss automatically sets the game system color to "Pal-M", like in the picture bellow:

![image](https://user-images.githubusercontent.com/107833265/177084676-1221bc41-1e19-43d1-83ee-647e285d3f3d.png)

> You can use Game Boy Interface or Enhanced mGBA with `--format=pal-m` to test this, and also `--format=ntsc-50` for the reverse problem. It's also possible to use Not64 with `VideoFormat = 2`.

You've nailed it! When I open the GBI via Swiss (always booting throgh Qoob Pro) in NTSC 480i or 480p, it works fine. When I set to "Pal-M", it blinks the screen really fast, and then... black screen, just with audio.

> Oh, PAL-M. I only have access to Euro-PAL cubes (and one US-NTSC), so that case was never tested.
> 
> I wonder why Swiss calls it DOL-002 though, isn't that the code for the power supply?

The Dol-002 is the code given to the brazilian consoles only (the same as the Power Supply, very good Nintendo). Just like the pic bellow:

![image](https://user-images.githubusercontent.com/107833265/177081679-c896b043-74c7-4bdd-a008-c9f432011610.png)

The Qoob Pro chip works fine on the console. I think it works because the modchip has it's own crystal oscillator, so I presume It doesn't use the one attached to the console's motherboard. Am I right, or gone too far?! Lol. I'll try to run Swiss through memory card's exploit (Twilight Princess Exploit) just to see how the console behaves (bypassing the Qoob Pro through the X button and booting with the original bios, first on my CRT, and then with HDMI).

![image](https://user-images.githubusercontent.com/107833265/177083356-e6f3c4e6-2e35-416d-8880-8eaa9539fbf9.png)

I don't know if it is going to be useful for you @ikorb and @Extrems, but I've dumped my console's IPL for you.
Please, use the link bellow:

https://1drv.ms/u/s!AsRv502u3B9niKpaEO2d8_dVHQ-4gA?e=wwNWYT

@ikorb, with these files, do you think it is possible to support Pal-M on GCVideo? If you think it is really not possible (as long as i'm in Brazil and unable to send the console to you, sorry...), maybe i'll try to convert the Pal-M to NTSC, just like people use to do here with with SNES, N64 and some other consoles that comes as NTSC and do not work with old Pal-M Tvs. But I really don't wanna change my console's motherboard at all.

Thank you guys for being so helpful!
